### DISCUSSION TITLE
What does it mean to be assigned something?

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [theforumhelpers/theforumhelpers.github.io#197] [Q&A] [answered]
why do we need to be assigned in the first place?

**Comment by leahcimto** [comment, marked as answer]
"The issue you are referring to is labelled as "needs-discussion". This means that people need to contribute ideas and a plan needs to be formed before a PR is created. Waiting to be assigned ensures that adequate discussion has taken place so that the person creating the PR understands what it is they should be changing. If anyone could come along and "pr freely", then this site would be a mishmash of so many different things that it would look and function terribly. Hence the process of:


1. Issue is created
2. Issue is discussed if needed
3. A plan for how could be implemented is made
4. Someone is assigned
5. They create a PR
6. It gets reviewed
7. Changes are made if necessary
8. The PR is merged"

– @Accio1 https://github.com/theforumhelpers/theforumhelpers.github.io/pull/176#issuecomment-840237994

**Reply by Steve0Greatness** [reply]
thanks! That helps!
